### ISSUE TITLE
Test a 'real-world' use case

### ISSUE THREAD

**Issue by f-dangel** [plutonium-239/memsave_torch#3]
Add a test that takes a neural network, converts its layers and sets `requires_grad=True` for only a subset of parameters. Then compute a forward pass and backward pass, make sure the gradients are correct.
